lib/capybara/session.rb

```diff
@@ -303,6 +303,10 @@ def go_forward
       driver.go_forward
     end
 
+    def send_keys(*args, **kw_args)
```

Comment:
Is there a way to share documentation with the `Node#send_keys` version, or would we need to declare comment docs here as well?

Reply:
Just link to Node#send_keys docs (See ...)